src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();
```

Comment:
You don`t need this field

Reply:
there are some mistake 
because i dont have this lines 
check this is inteligi and onlione in codespace

Reply:
there are some mistake 
because I don't have this lines in inteligi and also check in code-space
![Uploading Снимок экрана 2023-11-03 в 10.46.49 AM.png…]()